appveyor.yml

```diff
@@ -11,7 +11,7 @@ artifacts:
 deploy:
 - provider: NuGet
   api_key:
-    secure: kYR3BYzJm3wSFbFjPhgTzuDHHcE8ApoNUmHvJvunWZ39nyrqVk6J6srjzYVQ7/gX
+    secure: 45/vGyNCdoOvWSorcVX6qYM3oC/mCBj0CDRXNZP4twlIrBiZ9sKtKMdHwufm4ogS
```

Comment:
How does it work in terms of safety? Anyone can send their package now?

Reply:
that key should be one replaced by the encrypted one from appveyor during a build. 
@nblumhardt can u confirm?

Reply:
Appveyor has the private key to decrypt this at build time (it's base-64 AES-256 cyphertext).